src/test/java/org/jenkinsci/plugins/gitclient/WorkspaceWithRepo.java

```diff
@@ -39,7 +39,8 @@ private void createEnv(File gitDir, String gitImplName, TaskListener listener) t
         cliGitCommand = new CliGitCommand(gitClient);
     }
 
-    private GitClient setupGitClient(File gitFileDir, @NonNull String gitImplName, TaskListener listener) throws Exception {
+    private GitClient setupGitClient(File gitFileDir, @NonNull String gitImplName, TaskListener listener)
+            throws Exception {
```

Comment:
Please don't make white space changes in existing files.  It creates barriers for other pull requests.
```suggestion
    private GitClient setupGitClient(File gitFileDir, @NonNull String gitImplName, TaskListener listener) throws Exception {
```